src/app/utils/tests/TestUtils.spec.ts

```diff
@@ -389,23 +382,25 @@ export class ComponentTestUtils<T extends MyGameComponent> {
     }
     public expectElementToHaveClass(elementName: string, cssClass: string): void {
         const element: DebugElement = this.findElement(elementName);
-        expect(element).withContext(elementName + ' should exist').toBeTruthy();
-        const classAttribute: string = element.attributes.class;
-        expect(classAttribute).withContext(elementName + ' should have class attribute').toBeTruthy();
-        const elementClasses: string[] = classAttribute.split(' ').sort();
+        expect(element).withContext(elementName + ' should exist').not.toBeNull();
```

Comment:
Par contre si on expect qu'il existe, le composant peut pas être undefined si ?

Reply:
hmm oui si il existe il est pas undefined, et donc tu veux dire quoi par rapport au code ?

Reply:
Bah là, en passant de tobeTruthy à not.toBeNull on considère (selon les context) qu'exister et être undefined, c'est compatible :D

Reply:
ook, donc changé à .toBeTruthy !